operations/image_management/Configure_IMS_to_validate_rpms.md

```diff
@@ -68,14 +92,113 @@ Configuring the Image Management Service (IMS) to validate the GPG signatures of
     [...]
     ```
 
-1. Create a `Dockerfile` to create a new `cray-ims-kiwi-ng-opensuse-x86_64-builder` image.
+1. (`ncn-mw#`) (CSM 1.5.0 and later) Obtain a copy of the `armentry.sh` script from `cray-ims-kiwi-ng-opensuse-x86_64-builder`.
```

Comment:
Since this is going into the CSM 1.6 docs, isn't the "CSM 1.5.0 and later" part unnecessary?

Reply:
Backporting into 1.5, probably should to 1.4 as well since the original instructions didn't work. Maybe even 1.3.